view/src/client.rs

```diff
@@ -196,6 +196,7 @@ pub trait ViewClient {
     ) -> Result<BTreeMap<AddressIndex, BTreeMap<asset::Id, Vec<QuarantinedNoteRecord>>>> {
         let notes = self
             .quarantined_notes(pb::QuarantinedNotesRequest {
+                token: Option::None,
                 account_id: Some(account_id.into()),
             })
```

Comment:
Rather than explicitly setting `token: Option::None`, I think it'd be cleaner to do
```suggestion
            .quarantined_notes(pb::QuarantinedNotesRequest {
                account_id: Some(account_id.into()),
                ..Default::default()
            })
```
The advantage of this is that it's more clear to the reader that the `token` value is being left unset-as-a-default, rather than that we're explicitly setting it to `None` for some specific reason.